cassandra-plugins/src/test/java/co/cask/hydrator/plugin/test/ETLCassandraTest.java

```diff
@@ -360,7 +351,7 @@ private void testCassandraRealtimeSink() throws Exception {
       public Boolean call() throws Exception {
         CqlResult result = client.execute_cql3_query(ByteBufferUtil.bytes(cqlQuery),
                                                      Compression.NONE, ConsistencyLevel.ALL);
-        return result.rows.size() >= 2;
+        return result.rows.size() > 0;
```

Comment:
the only reason it waited for 2 before was to check that the timestamp columns were not equal (data generator source).  Since its been moved to mock source, we can just check the timestamp column directly.